<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClipboardOptionsWidget</class>
 <widget class="QWidget" name="ClipboardOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb">
     <property name="title">
      <string>&quot;Watch clipboard&quot; mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="clipboardRadio">
        <property name="text">
         <string>&amp;Clipboard</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="selectionPage">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="hl1">
            <item>
             <widget class="QRadioButton" name="selectionRadio">
              <property name="text">
               <string>&amp;Primary selection</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="l1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>after</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="selectionDelayBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="l2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>milliseconds</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hs">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="raiseWindowCheck">
            <property name="text">
             <string>&amp;Raise application window</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="v1">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>147</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="findBufferPage">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="findBufferRadio">
            <property name="text">
             <string>&amp;Find buffer</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="v2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>195</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>selectionRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectionDelayBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectionRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>l2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectionRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>l1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
